src/pytest/test_debug_commands.py

```diff
@@ -55,9 +75,35 @@ def testDumpTagIndexWrongArity(self):
     def testDumpUnexistsTagIndex(self):
         self.env.expect('FT.DEBUG', 'dump_tagidx', 'idx', 't1').raiseError()
 
+    def testDumpTagIndexInvalidKeyType(self):
+        self.env.expect('FT.DEBUG', 'dump_tagidx', 'foo', 't1').raiseError()
+
     def testDumpTagIndexInvalidSchema(self):
         self.env.expect('FT.DEBUG', 'dump_tagidx', 'idx1', 't').raiseError()
 
+    def testInfoTagIndex(self):
+        self.env.expect('FT.DEBUG', 'info_tagidx', 'idx', 't').equal(['num_values', 1L])
```

Comment:
@ashtul  pls add the skip cluster to all the functions here (and basically all the functions that uses debug command if it can not be tested in any other way)

Reply:
@MeirShpilraien `skipOnCluster` on `init` skip all tests for cluster

Reply:
:+1: